docs/Rapid/4-Keyper Manual/4-Adaptive Designer/6-How to configure connection with data table/6-How to configure connection with data table.md

```diff
@@ -0,0 +1,132 @@
+# How to configure connection with data table?
+
+Establishing a connection with a data table enables the system to fetch and / or feed data to the data table from the form.   
+
+:::info[Please note]
+In Adaptive Documents, there are two ways to levels of connections:  
+  1. at the document level - the table is connected to the document.
+  2. at the element level - the table column is connected to the element of the document.  
```

Comment:
Element level connections are only supported on legacy forms. They cannot be created in the v3 form designer.